core/crypto.mk

```diff
@@ -228,3 +228,83 @@ _CFG_CORE_LTC_HASH := $(call ltc-one-enabled, MD5 SHA1 SHA224 SHA256 SHA384 \
 _CFG_CORE_LTC_MAC := $(call ltc-one-enabled, HMAC CMAC CBC_MAC)
 _CFG_CORE_LTC_CBC := $(call ltc-one-enabled, CBC CBC_MAC)
 _CFG_CORE_LTC_ASN1 := $(call ltc-one-enabled, RSA DSA ECC)
+
+###############################################################
```

Comment:
@ldts , 
all those configs should be moved to the core/drivers/crypto/se050/ forlder now that you have one.
what do you think?

Reply:
@sdininno sounds good. let me try doing that.
side note, do you have an estimate for the ECC api?

Reply:
@sdininno does what I just pushed aligns with what you expected?

Reply:
anyway, it seems cleaner to me. @jforissier  I think we should be getting there to getting this merged?